frontend/src/metabase/query_builder/components/QueryModals.jsx

```diff
@@ -211,10 +217,15 @@ class QueryModals extends React.Component {
       <Modal onClose={onCloseModal}>
         <EntityCopyModal
           entityType="questions"
-          entityObject={this.props.card}
+          entityObject={{
```

Comment:
Might be cleaner if we declare an entityObject constant above this return, and have a more direct
`entityObject={entityObject}`
here.